build/common.sh

```diff
@@ -61,6 +61,8 @@ KUBE_CROSS_IMAGE="${KUBE_CROSS_IMAGE:-"${KUBE_BASE_IMAGE_REGISTRY}/kube-cross"}"
 readonly KUBE_CROSS_IMAGE
 KUBE_CROSS_VERSION="${KUBE_CROSS_VERSION:-"${KUBE_BUILD_IMAGE_CROSS_TAG}"}"
 readonly KUBE_CROSS_VERSION
+KUBE_CROSS_CONTAINER_ROOT="/go/src/k8s.io/kubernetes"
```

Comment:
nit:
```suggestion
KUBE_CONTAINER_ROOT="/go/src/k8s.io/kubernetes"
```

KUBE_CROSS is the image.

this nit isn't worth further delaying the CI fix though